CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Here is a template for new release sections
 
 ### Added
 - industrial process (#589)
+- Fuel-powered electricity generation and relations (#592)
```

Comment:
```suggestion
- fuel-powered electricity generation (#592)
```
I suggest mentioning only the class. Else one could think that we defined here new relations.

Reply:
Okay, I changed it now.